.travis.yml

```diff
@@ -29,18 +29,13 @@ matrix:
 
 before_install:
     # Download macOS specific extra dependencies.
-    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then git clone https://github.com/llvm-mirror/clang.git ~/llvm --branch master --single-branch --depth 1; fi
-    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then curl http://releases.llvm.org/3.9.0/clang+llvm-3.9.0-x86_64-apple-darwin.tar.xz -O; fi
-    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update;               fi
-    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install thrift@0.9;                fi
+    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
```

Comment:
What is the default clang version in the Travis OSX? Was there an update and we do not have to download it?

Reply:
Sorry if it caused any confusion. I used Travis about one and half years ago and don't know how was it updated. I read it somewhere and this is why I am tried it. Travis support did not answered it to me yet.

Reply:
@gyorb https://docs.travis-ci.com/user/reference/osx/#Compilers-and-Build-toolchain There isn't any Clang included in the macOS environment. It was the Ubuntu images that had Clang updated to 3.9 in September.

Reply:
@whisperity and in the xcode image? 9.0 & 9.1 is available

Reply:
You can always set up Travis for your own fork and make a custom `.travis.yml` file that tries to use some Clang installed on the system, and ask its version.

Though I fear it might be newer than the Clang we use in the Linux image, which can make testing tricky on bug hashes. (Although between 3.9 and 6.0-trunk, it didn't show any *actual* problems.)